lib/ansible/modules/cloud/azure/azure_rm_sqlserver_facts.py

```diff
@@ -0,0 +1,205 @@
+                    sample: sqlcrudtest-4645
+                type:
+                    description:
+                        - Resource type.
+                    returned: always
+                    type: str
+                    sample: Microsoft.Sql/servers
+                location:
+                    description:
+                        - Resource location.
+                    returned: always
+                    type: str
+                    sample: japaneast
+                kind:
+                    description:
+                        - Kind of sql server. This is metadata used for the Azure portal experience.
+                    returned: always
+                    type: str
+                    sample: v12.0
+                version:
+                    description:
+                        - The version of the server.
+                    returned: always
+                    type: str
+                    sample: 12.0
+                state:
+                    description:
+                        - The state of the server.
+                    returned: always
+                    type: str
+                    sample: Ready
+                fully_qualified_domain_name:
+                    description:
+                        - The fully qualified domain name of the server.
+                    returned: always
+                    type: str
+                    sample: fully_qualified_domain_name
+'''
+
+from ansible.module_utils.azure_rm_common import AzureRMModuleBase
+
+try:
+    from msrestazure.azure_exceptions import CloudError
+    from msrestazure.azure_operation import AzureOperationPoller
+    from azure.mgmt.sql import SqlManagementClient
+    from msrest.serialization import Model
+except ImportError:
+    # This is handled in azure_rm_common
+    pass
+
+
+class AzureRMServersFacts(AzureRMModuleBase):
+    def __init__(self):
+        # define user inputs into argument
+        self.module_arg_spec = dict(
+            resource_group=dict(
+                type='str',
+                required=True
+            ),
+            server_name=dict(
+                type='str'
+            )
+        )
+        # store the results of the module operation
+        self.results = dict(
+            changed=False,
+            ansible_facts=dict()
+        )
+        self.mgmt_client = None
+        self.resource_group = None
+        self.server_name = None
+        super(AzureRMServersFacts, self).__init__(self.module_arg_spec)
+
+    def exec_module(self, **kwargs):
+        for key in self.module_arg_spec:
+            setattr(self, key, kwargs[key])
+        self.mgmt_client = self.get_mgmt_svc_client(SqlManagementClient,
+                                                    base_url=self._cloud_environment.endpoints.resource_manager)
+
+        if (self.resource_group is not None and
+                self.server_name is not None):
+            self.results['sqlservers'] = self.get()
```

Comment:
The docs say this should be `servers`, I'm ok with either but it should be consistent.

Reply:
yes, sorry, made mistake, i will submit fix in a moment